web-client/src/providers/socketRouter.js

```diff
@@ -88,6 +88,11 @@ export const socketRouter = (app, onMessageCallbackFn) => {
           ...message,
         });
         break;
+      case 'serve_document_complete_error':
```

Comment:
Nit: Should this just be `serve_document_error`? As most likely service never completed if there was an error